src/tools/user-agent-parser/index.ts

```diff
@@ -0,0 +1,11 @@
+import { Browser } from '@vicons/tabler';
+import { defineTool } from '../tool';
+
+export const tool = defineTool({
+  name: 'User-agent parser',
+  path: '/user-agent-parser',
+  description: 'Detect and parse Browser, Engine, OS, CPU, and Device type/model from an user-agent string.',
+  keywords: ['user', 'agent', 'parser', 'browser', 'engine', 'os', 'cpu', 'device', 'user-agent', 'client'],
+  component: () => import('./user-agent-parser.vue'),
+  icon: Browser,
+});
```

Comment:
An it's new, the creation datein the component definition
```suggestion
  createdAt: new Date('2023-04-06'),
});
```